test/unit/property-test.js

```diff
@@ -236,7 +236,7 @@ lab.experiment('property - ', () => {
       'x',
       Joi.string().email({
         tlds: {
-          allow: ['example.com']
+          allow: ['com']
```

Comment:
This test was failing when I switched to joi. `example.com` isn't a top level domain.